syntax = "proto3";

package request;


message GetRequest {
    string filePath = 1;
   
}